lib/cc/analyzer/container.rb

```diff
@@ -116,7 +154,11 @@ def reap_running_container
       end
 
       def timeout
-        (ENV["CONTAINER_TIMEOUT_SECONDS"] || DEFAULT_TIMEOUT).to_i
+        ENV.fetch("CONTAINER_TIMEOUT_SECONDS", DEFAULT_TIMEOUT).to_i
```

Comment:
Hmm, could that happen? I'd think it could either be empty string (which would be 0) or a value, but I'm not sure how a `nil` value could be present in `ENV`.